avocado/plugins/multiplex.py

```diff
@@ -35,12 +35,10 @@ def __init__(self, *args, **kwargs):
         super(Multiplex, self).__init__(*args, **kwargs)
 
     def configure(self, parser):
-        if multiplexer.MULTIPLEX_CAPABLE is False:
-            return
-
         parser = super(Multiplex, self).configure(parser)
-        parser.add_argument('multiplex', nargs='+',
-                            help='Path(s) to a multiplex file(s)')
+        if multiplexer.MULTIPLEX_CAPABLE:
+            parser.add_argument("-m", '--multiplex', nargs='*',
```

Comment:
Even now we use multiple sources, so I'm pretty confident we can do that. First one is `--mux-inject`, which injects simple pairs into a specific paths. When `avocado-virt` is installed, it adds another set of values into specific locations. Last one currently supported is the `--multiplex` and that one does not parse all files at the same time, it parses each file and merges them together (only to be merged again to the default_avocado_params, which already contain data from `--mux-inject` and `avocado-vt`). The only problem is prioritization (which plugin is first) but I don't think it's urgent. We can say people can use it, but the order is undefined and if they want the order they should rewrite their configs into one implementation (use only one plugin).

I see the `Mux` as database, we have support for special flags to allow flexible merging. I don't expect people to use this often, but it should be possible.

As for the arguments I like the `--mux-yaml` and `--multiplex` with deprecation notice, I'll add it into the next version.